examples/jsm/geometries/TextGeometry.js

```diff
@@ -35,7 +36,15 @@ class TextGeometry extends ExtrudeGeometry {
 
 			// translate parameters to ExtrudeGeometry API
 
-			parameters.depth = parameters.height !== undefined ? parameters.height : 50;
+			if ( parameters.depth === undefined && parameters.height !== undefined ) {
```

Comment:
This PR does two things in one go. It adds support to the editor and renames the `height` parameter to `depth`. 

Ideally, there is a single PR for each change since both changes are independent of each other.

Besides, if `height` is renamed, the example code (meaning `webgl_geometry_text.html` and others) should be updated in order to avoid deprecation warnings.

That said, I also favor `depth` since `height` was a confusing name for describing the thickness (or depth) of the text.

Reply:
Sure I can do that. I'll wait on the confirmation of the `loader.registerGeometry()` api before making the changes.

Reply:
actually the change to extract is easier to do, so it's in https://github.com/mrdoob/three.js/pull/27949